client/src/utils/parser.js

```diff
@@ -9,7 +9,7 @@ function generateCodeForMultiple(bpmnTasks) {
     let currentTask = bpmnTasks[task];
     let rpaTaskName = currentTask['_attributes']['arkRPA:task'];
     let rpaTaskInputString = currentTask['_attributes']['arkRPA:inputVars'];
-    let rpaTaskInput = rpaTaskInputString.replaceAll('/', '');
+    let rpaTaskInput = rpaTaskInputString.replace(/[\/]/g, '');
```

Comment:
why was this change necessary?

Reply:
plain JS does not know pelaceAll, so this caused the tests to crash